page/page.njk

```diff
@@ -46,10 +47,24 @@
 {% endblock %}
 
 {% block header %}
-    {{govukHeader({
-        serviceName: "Claim criminal injuries compensation",
-        serviceUrl: "https://www.gov.uk/claim-compensation-criminal-injury/make-claim"
-    })}}
+    {% if loggedInUser %}
```

Comment:
rename to `isAuthenticated` or something more binary to represent a state. i.e. is this variable to hold the user ID, or to signify that they are logged in?